Sources/Orbit/Components/InputField.swift

```diff
@@ -160,15 +215,45 @@ public extension InputField {
     }
 }
 
+// MARK: - Model
+private class SecureViewModel: ObservableObject {
+
+    @Published var isShowSecuredText = false {
+        didSet {
+            if isShowSecuredText {
+
+                timerCancellation = Timer.publish(every: 3, on: .main, in: .default)
+                    .autoconnect().first()
+                    .sink { [weak self] _ in
+                        self?.isShowSecuredText = false
+                }
+            } else {
+                timerCancellation = nil
+            }
+        }
+    }
+
+    private var timerCancellation: AnyCancellable?
+
+    init() {  /* empty */ }
+}
+
 // MARK: - Previews
 struct InputFieldPreviews: PreviewProvider {
 
     static var previews: some View {
-        PreviewWrapper {
-            standalone
-            snapshots
+        Group {
+            PreviewWrapper {
+                standalone
+                snapshots
+            }
+            .previewLayout(PreviewLayout.sizeThatFits)
+            PreviewWrapper {
```

Comment:
I cant see a difference between these two grouped wrappers 🤔

Reply:
good catch, it's a duplication. Will fix